tests/__main__.py

```diff
@@ -112,9 +164,9 @@ def check_lint():
     print('...done')
 
 
-def run_tests(argv, env, coverage=False):
+def run_tests(argv, env, config):
```

Comment:
run_tests() doesn't need all of config, right?

Reply:
No. I admit to be a bit lazy here - will fix this up.